demo/index.html

```diff
@@ -221,32 +227,6 @@
       </demo-snippet>
 
       <h3>
-        Tabs can be used in a <code>paper-toolbar</code>.
```

Comment:
Ok, I can put a demo there. Are people just confused about what it does in general?